main.tf

```diff
@@ -0,0 +1,20 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.16"
+    }
+  }
+
+  required_version = ">= 1.2.0"
+}
+
+provider "aws" {
+  region  = "us-east-1"
+}
+
+resource "aws_instance" "app_server" {
+  ami           = "ami-0715c1897453cabd1"
+  instance_type = "t2.micro"
+
+}
```

Comment:
could you please, add a newline at the end of file.

Reply:
i dont understand when you say new line